deploy/templates/deployment.yaml

```diff
@@ -24,13 +24,37 @@ spec:
       nodeSelector:
         {{- toYaml .Values.nodeSelector | nindent 8 }}
       {{- end }}
-      {{- if .Values.affinity }}
       affinity:
+      {{- if .Values.affinity }}
         {{- toYaml .Values.affinity | nindent 8 }}
+      {{- else }}
```

Comment:
What's the default `nodeAffinity` used for?

Reply:
Prefer running the provider in system node, not running it in windows node and virtual kublet node.